lib/texture1.c

```diff
@@ -62,7 +51,7 @@ ktxTexture1_constructCommon(ktxTexture1* This)
     if (This->_private == NULL) {
         return KTX_OUT_OF_MEMORY;
     }
-	memset(This->_private, 0, sizeof(*This->_private));
+        memset(This->_private, 0, sizeof(*This->_private));
```

Comment:
wrong number of spaces in place of the tab. Should only be 4.

Reply:
The various indentation things you noted should be fixed now.